src/components/ReferredRecord/ReferredRecord.test.js

```diff
@@ -0,0 +1,35 @@
+import '../../../test/jest/__mock__';
+import React from 'react';
```

Comment:
Could you remove `import React from 'react';`?

Reply:
Removed